### PR TITLE
REFACTOR : refactor updateTaskStatus

### PR DESCRIPTION
**Pull request by sahsisunny**
### Issue: 
For fixing the below issue we need to refactor this code.
image

### Description: 
Refactoring `src/helperFunctions/updateTasksStatus.ts` file.

### Test Coverage
image

### Dev Tested:
- [x] Yes

### CODE REVIEW

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Review by bksh05 — APPROVED** [review]
LGTM

File: __tests__/Unit/Functions/updateTasksStatus.test.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        const expectedUpdatedTasks = [
+            { id: 1, status: 'IN_PROGRESS' },
+            { id: 2, status: 'IN_PROGRESS' },
+            { id: 3, status: 'ASSIGNED' },
+            { id: 4, status: 'AVAILABLE' },
+            { id: 5, status: 'COMPLETED' },
+            { id: 6, status: 'BLOCKED' },
+            { id: 7, status: 'unknown' },
+        ];
```
Comment (bharati-21) [inline __tests__/Unit/Functions/updateTasksStatus.test.ts:23]: Please move these to fixtures

**Review by endeavourmonk — COMMENTED** [review]
LGTM

**Pull request merged**